_tests/api/valid_features/woocommerce-product-feeds/woocommerce-product-feeds.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/*
+ * Plugin name: API - Log that the  New Product Editor is enabled.
+ */
+
+ add_action( 'woocommerce_admin_get_feature_config', static function( $features ) {
```

Comment:
This should be `add_filter`, returning `$features` at the end. That's why we are receiving the warnings on the test log

Reply:
Good catch!